src/Components/Publishing/__stories__/Article.story.tsx

```diff
@@ -59,6 +60,10 @@ ads.forEach(mediaType => {
   })
 })
 
+story.add("Basic Article", () => {
+  return <Article article={BasicArticle} />
+})
+
```

Comment:
Basic is a sub-category of feature article, but not an article type - we don't need this here if it is already included in the headers story.